python/lcatr/harness/job.py

```diff
@@ -232,6 +236,19 @@ def do_stage(self):
             util.log.error(msg)
             raise RuntimeError, msg
 
+        # Find out if we're in Prod database or not
+        try:
+            intRun = int(self.lims.runNumber)
```

Comment:
This line breaks the `fake_eT.py` environment, since it never assigns `self.lims.runNumber`.